Library/.rubocop.yml

```diff
@@ -24,6 +24,8 @@ AllCops:
     - "Homebrew/bin/*"
     - "Homebrew/vendor/**/*"
     - "Taps/*/*/vendor/**/*"
+    # vendored files
+    - "Homebrew/mechanize/http/content_disposition_parser.rb"
```

Comment:
Can we still have this in `Homebrew/vendor`?

Reply:
Sure, want to propose a specific path? (I'm not knowledgeable about the contract for `vendor` contents and various dependency management tools.)

Reply:
I guess just `Homebrew/vendor/mechanize` works.

Reply:
Or `Homebrew/vendor/gems/mechanize`.

Reply:
That makes sense to me, then we know that it's not ours to maintain.

Reply:
Done, PTAL